src/components/Contact.tsx

```diff
@@ -37,30 +31,23 @@ const Contact = () => {
   const fontSizes = [2, 3, 4, 5];
   return (
     <Flex flex="auto" alignItems="center">
-      <Container
-        textAlign={["center", "center", "center", "justify"]}
-        height="50%"
-      >
-        <ul>
-          <H1 fontSize={[4, 5, 6, 7]} py={3}>
-            {t("contact.header")}
-          </H1>
-          <H2 fontSize={fontSizes} py={3}>
-            {t("contact.subheader")}
-          </H2>
-          <ContactInfo fontSize={fontSizes} py={1}>
-            <a
-              href="mailto:ox@oxymore.com"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              {t("contact.email")}
-            </a>
-          </ContactInfo>
-          <ContactInfo fontSize={fontSizes}>
-            {t("contact.location")}
-          </ContactInfo>
-        </ul>
+      <Container textAlign="justify">
+        <H2 fontSize={[4, 5, 6, 7]} py={3}>
```

Comment:
Surely this is the H1? Title of the page

Reply:
yah, honestly I don't know what came over me, I started questioning everything about semantic html. think it was time to step away from the code at this point